dynawo/sources/Models/CPP/ModelNetwork/DYNModelLoad.cpp

```diff
@@ -192,14 +185,14 @@ ModelLoad::setFequations(std::map<int, std::string>& fEquationIndex) {
   unsigned int index = 0;
 
   if (isRestorative_) {
-    if (!TpIsZero_ && isRunning())
+    if (isRunning() && !TpIsZero_)
```

Comment:
Testing a boolean is cheaper than calling a method, this is why it was in second

Reply:
Hehe two different ways of considering the problem ;)

- My approach was functional: as of today, we got several loads not running() and Tp will never be zero in practical operational case. So I thought it was better this way. 

- Nevertheless, with improvements discussed on the network to be handled, this shouldn't be the case any longer (or at least not with such a difference in possibilities). Thus I can modify again if you want (I think that the method is an inlining of two booleans attributes no?)

Reply:
It is an inlining of an equality test (connectionState_ == CLOSED) and a conversion of a z value to a bool, so I think it would be better to keep it the former way

Reply:
Solved